source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/show/index.tsx

```diff
@@ -3,9 +3,10 @@ import { compose } from 'recompose';
 import { withTranslations, i18nProps } from '@lib/i18n';
 import { withData } from '../edit/with-data';
 import { UserAttributes } from '@data/models/user';
-import { getPictureUrl } from '@lib/auth0';
 import { attributesFor } from '@data';
 
+import md5 from 'md5-hash';
```

Comment:
I wish there were more rules for imports, but generally, I like to have thing sorted like this:
```
npm-package imports

top-level imports, sorted alphabetically

relative imports, sorted alphabetically
```
(with line breaks separating each section)